imports/ui/ranking/Ranking.html

```diff
@@ -7,7 +7,7 @@
             <th class="elo">ELO</th>
             <th class="games-played">Games Played</th>
         </tr>
-        {{ #each player in (getPlayers sortBy) }}
+        {{ #each player in (getPlayers sortBy format) }}
```

Comment:
All the code that's required here is literally not even 100 lines long, though. I get that it would be useful, but is reading the less than 60 lines of code which is just taking values and manipulating them something that requires commenting?